scattr/workflow/rules/freesurfer.smk

```diff
@@ -54,17 +58,23 @@ rule cp_fs_tsv:
         "cp -v {input.fs_tsv} {output.fs_tsv}"
 
 
+# Update wildcards here to include both subject wildcards
```

Comment:
Is this a TODO? Looks like the params have the wildcards

Reply:
Was TODO when I was having trouble getting just the subject and session wildcards passed for freesurfer directory structure. Comment will be removed